src/locale/locales/de/messages.po

```diff
@@ -1166,48 +1108,48 @@ msgstr "Erledigt{extraText}"
 
 #: src/view/com/auth/login/ChooseAccountForm.tsx:45
 msgid "Double tap to sign in"
-msgstr ""
+msgstr "Doppeltippen zum Anmelden"
 
 #: src/view/screens/Settings/index.tsx:755
 msgid "Download Bluesky account data (repository)"
-msgstr ""
+msgstr "Laden Sie Ihre Bluesky-Kontodaten herunter (Repository)"
 
 #: src/view/screens/Settings/ExportCarDialog.tsx:59
 #: src/view/screens/Settings/ExportCarDialog.tsx:63
 msgid "Download CAR file"
-msgstr ""
+msgstr "CAR-Datei herunterladen"
 
 #: src/view/com/composer/text-input/TextInput.web.tsx:247
 msgid "Drop to add images"
-msgstr ""
+msgstr "Drop zum Hinzufügen von Bildern"
```

Comment:
maybe someone could find a better word than 'drop'

Reply:
`Ablegen` would be a better one.

```suggestion
msgstr "Ablegen zum Hinzufügen von Bildern"
```